### PR TITLE
Fixed #18 Removed animations by adding prefers-reduced-motion

### LINKED ISSUE #18: [Cub] Remove animations for `prefers-reduced-motion: reduced;`
**Description**
Cub's Contraption has minor animations. Animations can sometimes promote certain types of migraines. If the user has disabled system animations on their device, the website should remove the animations too.

**Possible solution(s)**
The prefers-reduced-motion CSS media feature is used to detect if the user has requested that the system minimize the amount of non-essential motion it uses.
```css
@media (prefers-reduced-motion) {
  body {
    animation: none!important;
    transition: none!important;
  }
}
```

### PR DESCRIPTION
**Pull request by moody2times**
This pull request fixes #18 

I have added a new media query (prefers-reduced-motion: reduce) and inside it I removed all transition effects using the universal selector. There are no animation properties in the code, so I wrote only `transition: none`  for a cleaner code. I use a windows 10 pc and turned off SHOW ANIMATIONS IN WINDOWS and tested the code. The transition effects were disabled but were enabled when I turned it back on. The transform properties such as scale are still active, though as they are not within the scope of animations. 

In the css declaration under the prefers-reduced-motion, I used the `!important` keyword to give this rule the highest specificity because media queries have no specificity while the universal selector * has low specificity. The code would not work otherwise.

### CODE REVIEW

**Comment by mmaismma** [comment]
This does not solve the problem. The problem is not about removing the CSS property `transition`. **Any animation (English word) which changes the scale of an element can cause sickness, no matter if they are smooth or not.**

To fix the issue you need to remove the scaling of elements on `:hover` and `:active`. Removing them will solve the problem but replacing them with color-changing transitions will do even better.

*If you add color-changing transitions, make sure to make it very subtle i.e. don't change the color too much.*

**Review by mmaismma — CHANGES_REQUESTED** [review]
To fix the issue you need to remove the scaling of elements on `:hover` and `:active`. Removing them will solve the problem but replacing them with color-changing transitions will do even better.

**Comment by moody2times** [comment]
> 
> 
> This does not solve the problem. The problem is not about removing the CSS property `transition`. **Any animation (English word) which changes the scale of an element can cause sickness, no matter if they are smooth or not.**
> 
> To fix the issue you need to remove the scaling of elements on `:hover` and `:active`. Removing them will solve the problem but replacing them with color-changing transitions will do even better.
> 
> _If you add color-changing transitions, make sure to make it very subtle i.e. don't change the color too much._

Thanks for the feedback, I really appreciate it. To be honest, prefers-reduced-motion is a new territory for me and despite my research, it was not clear in my mind what kind of motion should be removed. But now with your guidance, I will remove the transforms as well and replace them with the suggested color-changing transitions. As for the colors, I will simply do a subtle shade of the existing color just like the example on the MDN docs.

Thanks for your patience 🙂.

**Comment by moody2times** [comment]
Changes have been implemented. All animations in the sense of the English word have been removed this time. As for the color-changing transitions, I used the same `background-color` used for the download button on `:hover` state to define a `border-color` and `color` for the .area elements `:hover` and `:active` states. It is very subtle so I hope it meets your expectation. If not, request for changes and I will work on it ASAP.

File: Source/cubfan135/style.css [hunk 1]
```diff
@@ -341,3 +341,17 @@ body.dark .area::placeholder {
...
 }
+
+@media screen and (prefers-reduced-motion: reduce) {
+  .area:active,
+  .area:hover{
+    transform: none !important;
+  }
+
+  /* style for the .area class elements except the download button */
+  .area:active:not(#downloader),
```
Comment (mmaismma) [inline Source/cubfan135/style.css]: On `:active` state all elements get a sepia filter (from line 341), so this line is not needed. It won't do much but I just pointed out.

Comment (mmaismma) [reply Source/cubfan135/style.css]: This is a very small change, so I'll change it myself and accept the PR.

Comment (moody2times) [reply Source/cubfan135/style.css]: You are right, there is a color transition on the `:focus` state which makes having the style on the `:active` state redundant. Thanks for pointing that out.

**Pull request merged**